src/task/index.html

```diff
@@ -106,6 +106,19 @@
             <option value="ASSIGNED">Assigned</option>
           </select>
         </div>
+        <!-- Options for AVAILABLE -->
+        <div class="inputBox" id="estimatedNoOfDaysInput">
+          <label for="estimatedNoOfDays">Estimated no. of days</label>
+          <input
```

Comment:
Will the estimatedNoOfDays be optional or it is the required field?

Reply:
Optional.

Reply:
Ok.